src/components/Diagram.tsx

```diff
@@ -137,6 +145,11 @@ const DiagramSpace = styled.svg`
     fill: ${DIAGRAM_HOVER_COLOR};
   }
 
+  g.table-row.table-row-measure:not(.table-row-selected):hover rect {
```

Comment:
This line should actually be:
```
  g.table-row.table-row-measure:not(.table-row-selected):hover > rect,
  g.table-row.table-row-measure:not(.table-row-selected):hover > path.table-row {
  ```
  So that it more precisely selects the rect on hover, and so that it selects the "bottom cap" of tables, which is a path not a rect. This is an oversight on my side, let's merge this in and then I'll correct.